nox/virtualenv.py

```diff
@@ -39,6 +39,22 @@
 _SYSTEM = platform.system()
 
 
+def find_uv() -> str | None:
+    uv = shutil.which("uv")
+    if uv is not None:
+        return uv
+
+    # Handle `pipx install nox[uv]` by looking for uv next to python.
+    uv = os.path.join(os.path.dirname(sys.executable), "uv")
+    if os.path.exists(uv):
+        return uv
+
+    return None
+
+
+UV = find_uv()
```

Comment:
Should we make sure this path is on the path? I'd expect `run("uv", ...)` to always work if the uv backend is used?

Reply:
Yeah, I also wondered about this...

My first intuition here is that we don't want this. If the user wants to run `uv` as a non-external command, they can `session.install("uv")` and it'll just work.

The problem is you can't add uv to the PATH without also adding every other command from the environment in which Nox and uv are installed. That seems prone to cause issues.

Reply:
Ideally, the env shouldn't contain any Python dependencies not explicitly `session.install`ed into it, and that includes uv's Python distribution.